src/views/Proposals.vue

```diff
@@ -150,10 +157,19 @@ export default {
     }
   },
   methods: {
-    ...mapActions(['getProposals'])
+    ...mapActions(['getProposals']),
+    selectTabState(tab) {
+      const tabSelected = tab ? tab : this.$route.params.tab;
+      if (tabSelected) {
```

Comment:
These 5 lines of code could be changed to one like this:
```
this.selectedState = tabSelected || this.namespace.defaultView
```

Reply:
good idea